api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.4.x.xml

```diff
@@ -28,14 +28,85 @@
+		<modifySql dbms="mssql">
+			<replace replace="CHAR(38)" with="UNIQUEIDENTIFIER DEFAULT NEWSEQUENTIALID()"/>
+		</modifySql>
+	</changeSet>
+	
+	<changeSet id="20200723-TRUNK-5410" author="tendomart">
+		<comment>Creating order_group_attribute table</comment>
+		<createTable tableName="order_group_attribute">
+			<column name="order_group_attribute_id" type="int" autoIncrement="true">
+				<constraints primaryKey="true" nullable="false" />
+			</column>
+			<column name="order_group_id" type="int"><constraints nullable="false" /></column>
+			<column name="attribute_type_id" type="int"><constraints nullable="false" /></column>
+			<column name="value_reference" type="text"><constraints nullable="false" /></column>
+			<column name="uuid" type="char(38)"><constraints nullable="false" unique="true" /></column>
+			<column name="creator" type="int"><constraints nullable="false"/></column>
+			<column name="date_created" type="datetime"><constraints nullable="false"/></column>
+			<column name="changed_by" type="int"/>
+			<column name="date_changed" type="datetime" />
+			<column name="voided" type="boolean" defaultValueBoolean="false">
+				<constraints nullable="false"/>
+			</column>
+			<column name="voided_by" type="int" />
+			<column name="date_voided" type="datetime" />
+			<column name="void_reason" type="varchar(255)" defaultValue="null"/>
+		</createTable>
+		<addForeignKeyConstraint constraintName="order_group_attribute_order_group_fk" baseTableName="order_group_attribute" baseColumnNames="order_group_id" referencedTableName="order_group" referencedColumnNames="order_group_id" />
+		<addForeignKeyConstraint constraintName="order_group_attribute_attribute_type_id_fk" baseTableName="order_group_attribute" baseColumnNames="attribute_type_id" referencedTableName="order_group_attribute_type" referencedColumnNames="order_group_attribute_type_id" />
+		<addForeignKeyConstraint constraintName="order_group_attribute_creator_fk" baseTableName="order_group_attribute" baseColumnNames="creator" referencedTableName="users" referencedColumnNames="user_id" />
+		<addForeignKeyConstraint constraintName="order_group_attribute_changed_by_fk" baseTableName="order_group_attribute" baseColumnNames="changed_by" referencedTableName="users" referencedColumnNames="user_id" />
+		<addForeignKeyConstraint constraintName="order_group_attribute_voided_by_fk" baseTableName="order_group_attribute" baseColumnNames="voided_by" referencedTableName="users" referencedColumnNames="user_id" />
+	</changeSet>
 
 	<changeSet id="20190815-Trunk-5412" author="gitacliff">
 		<preConditions onFail="MARK_RAN">
 			<not>
 				<columnExists tableName="order_group" columnName="order_group_reason"/>				
 			</not>
 		</preConditions>
-		<comment>Adding "order_group_reason" column to order_group table</comment>
```

Comment:
But the precondition says order_group_reason instead of previous_order_group

Reply:
@dkayiwa  what do you mean here ? this is Cliff's changeset.

Reply:
Then why did you change it?

Reply:
I think it was a mistake which i later changed.

Reply:
Alright.